lti_consumer/models.py

```diff
@@ -60,52 +62,89 @@ class LtiConfiguration(models.Model):
     # stored on the block, but should be expanded to
     # enable storing LTI configuration in this model.
     CONFIG_ON_XBLOCK = 'CONFIG_ON_XBLOCK'
+    CONFIG_ON_DB = 'CONFIG_ON_DB'
     CONFIG_STORE_CHOICES = [
-        (CONFIG_ON_XBLOCK, 'Configuration Stored on XBlock fields'),
+        (CONFIG_ON_XBLOCK, _('Configuration Stored on XBlock fields')),
+        (CONFIG_ON_DB, _('Configuration Stored on this model')),
     ]
     config_store = models.CharField(
         max_length=255,
         choices=CONFIG_STORE_CHOICES,
         default=CONFIG_ON_XBLOCK,
     )
 
+    # A secondary ID for this configuration that can be used in URLs without leaking primary id.
```

Comment:
What is the primary ID now?

Reply:
The original default auto-generated ID is till the primary ID.